laws/src/main/scala/cats/laws/discipline/TraverseTests.scala

```diff
@@ -80,7 +85,11 @@ trait TraverseTests[F[_]] extends FunctorTests[F] with FoldableTests[F] with Uno
           "traverse ref mapAccumulate" -> forAll(laws.mapAccumulateRef[M, A, C] _),
           "traverse ref mapWithIndex" -> forAll(laws.mapWithIndexRef[A, C] _),
           "traverse ref traverseWithIndexM" -> forAll(laws.traverseWithIndexMRef[Option, A, C] _),
-          "traverse ref zipWithIndex" -> forAll(laws.zipWithIndexRef[A, C] _)
+          "traverse ref zipWithIndex" -> forAll(laws.zipWithIndexRef[A, C] _),
+          "traverse ref zipWithLongIndex" -> forAll(laws.zipWithLongIndexRef[A, C] _),
+          "traverse ref mapWithLongIndex" -> forAll(laws.mapWithLongIndexRef[A, C] _),
+          "traverse ref traverseWithLongIndexM" -> forAll(laws.traverseWithLongIndexMRef[Option, A, C] _),
+          "traverse ref updated" -> forAll(laws.updatedRef[A, A](_, _, _))
```

Comment:
Huh, the naming of these law tests seems to be inconsistent in Cats 🤔 
https://github.com/typelevel/cats/blob/83ad02b525946c7eb6f74205f6c1968a08a6e75b/laws/src/main/scala/cats/laws/discipline/FoldableTests.scala#L60-L67